### PR TITLE
create path to individual presentation pages

### PR DESCRIPTION
**Pull request by susannnnxiao**
1. go into presentations.json and copy an _id from an object
2. paste id after /presentations/

this should pull up a page that lists the _id!

### CODE REVIEW

**Comment by susannnnxiao** [comment]
i haven't gotten to edge case testing yet. so i am not sure what inputs would break it

File: src/server.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import express from "express";
 import handlebars from "handlebars";
 import fs from "fs";
-import presentations from './db/presentations.json' with {type: "json"}
+import presentations from './db/presentations.json' assert {type: "json"}
```
Comment (mcMickJuice) [inline src/server.js:4]: Why the change to assert here? Will an error occur if the file you're importing is not json?

File: src/server.js [hunk 2]
```diff
@@ -65,6 +65,14 @@ app.get("/presentations", (_, res) => {
   res.status(200).send(html);
 });
 
+app.get("/presentations/:id", (req, res) => { //the id path
+  const presID = req.params.id;
+  const presentation = presentations.find((presentation) => presentation._id == presID);
+  console.log(presentation);
```
Comment (mcMickJuice) [inline src/server.js:71]: Let's remove this console log

File: src/server.js [hunk 3]
```diff
@@ -65,6 +65,14 @@ app.get("/presentations", (_, res) => {
   res.status(200).send(html);
 });
 
+app.get("/presentations/:id", (req, res) => { //the id path
+  const presID = req.params.id;
+  const presentation = presentations.find((presentation) => presentation._id == presID);
```
Comment (mcMickJuice) [inline src/server.js:70]: In the event that presentation is not found, let's return a 404.

File: src/templates/index.hbs [hunk 4]
```diff
@@ -7,7 +7,9 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
   </head>
   <body>
-    {{> favorite presentation='My First Presentation' }}
+    <div>
+      {{test}}
```
Comment (mcMickJuice) [inline src/templates/index.hbs:11]: Can you revert this change? It looks like you were testing things out here

File: src/server.js [hunk 5]
```diff
@@ -65,6 +65,14 @@ app.get("/presentations", (_, res) => {
...
+  console.log(presentation);
+  const html = render("presentationDesc", {testing: presentation});
```
Comment (mcMickJuice) [inline src/server.js:72]: Let's replace `testing` with `presentation`, and update the template as well.

**Review by mcMickJuice — CHANGES_REQUESTED** [review]
The code looks wired up well!  Just some cleanup needed.

File: src/server.js (hunk 1 above)
Comment (susannnnxiao) [reply src/server.js:4]: i think it wasn't working for me unless I has assert, might be a versioning issue. I can change it back!

File: src/templates/index.hbs (hunk 4 above)
Comment (susannnnxiao) [reply src/templates/index.hbs:11]: yes my bad!

File: src/server.js (hunk 5 above)
Comment (susannnnxiao) [reply src/server.js:72]: sounds good! thanks for the feedback!
